debian/rules

```diff
@@ -31,6 +31,7 @@ extraopts += -DCMAKE_INSTALL_LIBDIR=/usr/lib
 extraopts += -DCMAKE_INSTALL_LIBEXECDIR=/usr/libexec
 extraopts += -DCMAKE_INSTALL_SYSCONFDIR=/etc
 extraopts += -DCMAKE_INSTALL_SYSTEMD_SERVICEDIR=/lib/systemd/system
+extraopts += -DCMAKE_INSTALL_SYSTEMD_GENERATORDIR=/lib/systemd/system-generators
```

Comment:
@huww98 could you please reconsider my suggestion at https://github.com/ceph/ceph/pull/40844/commits/b7e57a079ac0ba387b8c2ec7ad93df37c02fd90d#r613158301 ?